http/handler_store.go

```diff
@@ -747,5 +831,7 @@ func (h *storeHandler) bindRoutes(router *Router) {
 	router.AddRoute("/debug/dump", http.MethodGet, debugDump, h.PrintDump)
 	router.AddRoute("/schema", http.MethodPost, addSchema, h.AddSchema)
 	router.AddRoute("/lens", http.MethodPost, setMigration, h.SetMigration)
```

Comment:
suggestion:  Unsure whether this should be a todo or not...

We probably want to move the `SetMigration` http path before v1.  And it's current path is where `AddLens` should probably exist (bringing it into scope of this PR).  This way `Add` (POST) and `Get` (GET) live in the same location, then the 'special' `SetMigration` lives somewhere else, maybe even somewhere under `/collection`.